.github/workflows/ci.yml

```diff
@@ -187,13 +208,15 @@ jobs:
       fail-fast: false
       matrix:
         os:
-          - [self-hosted, linux, arm64]
-          - macos-latest
-          - [self-hosted, macos, arm64]
           - windows-latest
```

Comment:
You could remove the matrix completely and just use `runs-on: windows-latest`. Also, at this point all the special steps for ubuntu and mac can be removed.

Reply:
I left them there so if we want to disable runners again we just have to move the elements from one list to the other.

Eventually I want to move windows to the `unittests` job and completely remove `unittest-slower`

Reply:
I understand the motivation, but I think this is a bad idea. If the stuff there is not actually used, there's a high risk it will get outdated pretty soon. IMO, we shouldn't just keep unused stuff around "just in case it is needed in the future".

Reply:
Added it as info to https://github.com/spacemeshos/go-spacemesh/issues/4599 and assigned myself.